ADRs/RH/SIG-SRE/ADR-00005 SLO Lifecycle.md

```diff
@@ -0,0 +1,131 @@
+# ADR-00000 Template
+
+Authors: Lisa Seelye <@lisa>, Jeremy Eder
+
+## Status
+
+Draft
+
+## Problem Statement
+
+There are a number of complex processes and [personas][personas-adr] that play different roles in the successful implementation of an SLO. One hurdle some teams encounter with creating SLOs for their services is the absence of a guide to show the steps for getting started.
+
+This document aims to provide a base on top of which teams can build a successful SLO implementation.
+
+
+## Goals
+
+* The guide is generic enough to be used by any service as-is.
+* The guide doesn't prevent anyone from taking it and tweaking it to their needs.
+
+## Non-goals
+
+
+## Current Architecture
+
+* Various ad-hoc practices, where they already exist.
+
+## Proposed Architecture
+
+Add Service Level Objectives (SLOs) as a feature to services with its own distinct lifecycle described herein. The PM will be accountable for the SLO feature.
```

Comment:
We got some feedback that using the word feature would not work out the way I thought it would, recommend not using this word.

Reply:
Good catch. I will make the change.